lib/webdriveragent.js

```diff
@@ -25,6 +24,37 @@ const WDA_AGENT_PORT = 8100;
 const WDA_BASE_URL = 'http://localhost';
 const BUILD_TEST_DELAY = 1000;
 
+
+class NoSessionProxy extends JWProxy {
```

Comment:
Like so should work: https://gist.github.com/imurchie/c9e0f9136d922db537cd87a4be66d8b9